src/config/Config.ts

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2021-2022 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { DEFAULT, IConfigOptions } from "./ConfigOptions";
+
+export class Config {
+  static instance: Config;
```

Comment:
Could this be a getter that throws when the internal instance is undefined, i.e. `init()` has not been called yet